### PR TITLE
Navbar feed back

### PR DESCRIPTION
**Pull request by neelxie**
#### What this PR does
- Fix alignment and color for navbar
- Implement feedback given for changes to the navbar

### CODE REVIEW

**Comment by mwotil** [comment]
As @StephenAraka pointed out, lets have consistency in the design colours. Additionally, could you push the profile to the extreme right. There is too much space after it

**Comment by neelxie** [comment]
> Use the exact same color as in the design...👇
> 
> Screenshot from 2020-02-17 14-10-46

@StephenAraka thank you for the feedback, I kindly request you take time to check out the colors used against the colors of the design and point out what I am missing, stop alleging and prove that am actually using a different color from the design, take time to look at the code as well, will wait for your feedback when you do.

**Comment by mwotil** [comment]
> > Use the exact same color as in the design...👇
> > Screenshot from 2020-02-17 14-10-46
> 
> @StephenAraka thank you for the feedback, I kindly request you take time to check out the colors used against the colors of the design and point out what I am missing, stop alleging and prove that am actually using a different color from the design, take time to look at the code as well, will wait for your feedback when you do.

So which colour is which based on the two that @StephenAraka attached?

**Comment by mwotil** [comment]
> > > Use the exact same color as in the design...👇
> > > Screenshot from 2020-02-17 14-10-46
> > 
> > 
> > @StephenAraka thank you for the feedback, I kindly request you take time to check out the colors used against the colors of the design and point out what I am missing, stop alleging and prove that am actually using a different color from the design, take time to look at the code as well, will wait for your feedback when you do.
> 
> So which colour is which based on the two that @StephenAraka attached?

Home Page

**Comment by neelxie** [comment]
> > > Use the exact same color as in the design...👇
> > > Screenshot from 2020-02-17 14-10-46
> > 
> > 
> > @StephenAraka thank you for the feedback, I kindly request you take time to check out the colors used against the colors of the design and point out what I am missing, stop alleging and prove that am actually using a different color from the design, take time to look at the code as well, will wait for your feedback when you do.
> 
> So which colour is which based on the two that @StephenAraka attached?

@mwotil a comprehensive review requires you run and check the code against the design, my code is available and so is the design, if my color is wrong point it out in the code and I fix it. @mwotil you feedback as well has been worked on, kindly do check it out.

**Comment by mwotil** [comment]
> > > > Use the exact same color as in the design...👇
> > > > Screenshot from 2020-02-17 14-10-46
> > > 
> > > 
> > > @StephenAraka thank you for the feedback, I kindly request you take time to check out the colors used against the colors of the design and point out what I am missing, stop alleging and prove that am actually using a different color from the design, take time to look at the code as well, will wait for your feedback when you do.
> > 
> > 
> > So which colour is which based on the two that @StephenAraka attached?
> 
> @mwotil a comprehensive review requires you run and check the code against the design, my code is available and so is the design, if my color is wrong point it out in the code and I fix it. @mwotil you feedback as well has been worked on, kindly do check it out.

My environment is not set up, hopefully can do that later today but in the meantime you can post an image

**Comment by neelxie** [comment]
> > > > > Use the exact same color as in the design...👇
> > > > > Screenshot from 2020-02-17 14-10-46
> > > > 
> > > > 
> > > > @StephenAraka thank you for the feedback, I kindly request you take time to check out the colors used against the colors of the design and point out what I am missing, stop alleging and prove that am actually using a different color from the design, take time to look at the code as well, will wait for your feedback when you do.
> > > 
> > > 
> > > So which colour is which based on the two that @StephenAraka attached?
> > 
> > 
> > @mwotil a comprehensive review requires you run and check the code against the design, my code is available and so is the design, if my color is wrong point it out in the code and I fix it. @mwotil you feedback as well has been worked on, kindly do check it out.
> 
> My environment is not set up, hopefully can do that later today but in the meantime you can post an image

@mwotil here it is
image

**Comment by mwotil** [comment]
You can push it further @neelxie. You can get some inspiration from the Github one.

**Comment by neelxie** [comment]
> You can push it further @neelxie. You can get some inspiration from the Github one.

@mwotil done.

**Comment by Mubangizi** [comment]
These colors look different @StephenAraka  like from that screenshot
but on checking i see there similar
Screenshot from 2020-02-18 11-27-10

**Comment by Mubangizi** [comment]
Screenshot from 2020-02-18 11-27-10
#008AC1

File: src/components/NavBar/NavBar.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .Navigation{
-    background-color: #99D2E9;
+    background-color: #008ac1;
```
Comment (Mubangizi) [inline src/components/NavBar/NavBar.css]: Hahaha You just changed @neelxie

**Comment by neelxie** [comment]
> Good job... Just make the arrow on the right slightly smaller!

@StephenAraka thank you. this has been worked on.

**Pull request merged**